docker-compose.yml

```diff
@@ -28,7 +28,7 @@ services:
       context: .
       dockerfile: hosted/Dockerfile
       cache_from:
-        - "${DOCKER_REPO}corpora-explorer:branch-main"
+        - "${DOCKER_REPO}corpora-explorer:latest"
```

Comment:
Any reason for this?

Reply:
The tag on the images is "latest," not "branch-main," so when it tried to pull from cache, it often errored that it couldn't find it. Switching to "latest" fixes that.

Reply:
Ah sorry, this is just for caching. 👍 then